rules/windows/execution_suspicious_short_program_name.toml

```diff
@@ -0,0 +1,28 @@
+[metadata]
+creation_date = "2020/11/15"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/15"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies process execution with one char length process name. This is often done by adversaries while staging or
+executing temporary utilities.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Suspicious Execution - Short Program Name"
+risk_score = 47
+rule_id = "17c7f6a5-5bc9-4e1f-92bf-13632d24384d"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Execution"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and length(process.name) > 0 and
+ length(process.name) == 5 and host.os.name == "Windows" and length(process.pe.original_file_name) > 5
+'''
```

Comment:
Is this looking for a process name with a character length between 1 and 5? Should the operator in this expression `length(process.name) == 5` be changed to `>=` ?

Reply:
looking for names like a.exe, b.exe etc. and pe.original_file_name length is > 5 (something.exe), example: 
![image](https://user-images.githubusercontent.com/64742097/99319163-5a80ce00-2869-11eb-9534-1cefdf206da8.png)

Reply:
Perfecto